source/index.js

```diff
@@ -155,6 +155,7 @@ export { default as pair } from './pair';
 export { default as partial } from './partial';
 export { default as partialRight } from './partialRight';
 export { default as partition } from './partition';
+export { default as splitWhenever } from './splitWhenever';
```

Comment:
This should be moved to the proper place alphabetically, right after `splitWhen`